website/src/utils/search.ts

```diff
@@ -184,6 +184,7 @@ export const getFieldValuesFromQuery = (
     hiddenFieldValues: FieldValues,
     schema: Schema,
 ): FieldValues => {
+    // here the hidden field values are there as the default - maybe if we make them settable, they will just override?
```

Comment:
```suggestion
```

maybe?

Reply:
Oh I think this was a note I left to myself 😅  I'll delete it.

Reply:
https://github.com/loculus-project/loculus/pull/3809/commits/32e73d368b190d9d5e5f23f362030a28963d8e88